src/components/custom/dashboard/client/pages/diet-tracker/MealOptionModal.jsx

```
@@ -0,0 +1,211 @@
+            return (
+              <div
+                key={index}
+                className={`relative p-4 rounded-lg border cursor-pointer transition-all duration-200 ${
+                  isSelected
+                    ? "border-[#FF6B00] bg-[#FF6B00]/5 ring-2 ring-[#FF6B00]/30"
+                    : isCurrentOption
+                    ? "border-green-500/50 bg-green-500/5"
+                    : "border-zinc-700 bg-zinc-800/30 hover:border-zinc-600 hover:bg-zinc-800/50"
+                }`}
+                onClick={() => setSelectedOption(option)}
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)